html-css/week1/index.html

```diff
@@ -0,0 +1,166 @@
+    <main>
+      <section class="about" id="about">
+        <div class="about-title">
+          <h2 class="about-title-text">About Me</h2>
+        </div>
+        <div class="about-text">
+          <p>
+            Passionate about web development with a strong foundation in
+            industrial design and extensive experience in ergonomics, user
+            interaction, and usability.<br /><br />Eager to leverage skills in
+            HTML, CSS, JavaScript, and React.js to create engaging and
+            user-friendly web applications.<br /><br />Experienced in applying
+            Scrum in project management and dedicated to continuous learning and
+            professional growth in front-end development.
+          </p>
+        </div>
+      </section>
+      <section class="skills" id="skills">
+        <div class="skills-container">
+          <!-- Left Side Skills (Yellow Lines) -->
+          <ul class="skills-list left">
+            <li><span>HTML</span></li>
+            <li><span>CSS</span></li>
+            <li><span>JavaScript</span></li>
+            <li><span>TypeScript</span></li>
+            <li><span>React.js</span></li>
+            <li><span>Angular</span></li>
+            <li><span>VS Code</span></li>
+            <li><span>Git</span></li>
+          </ul>
+          <div class="skills-core">Technical Skills</div>
+          <ul class="skills-list right">
+            <li><span>Figma</span></li>
+            <li><span>Adobe Creative Suite</span></li>
+            <li><span>Smoke Testing</span></li>
+            <li><span>Functional Testing</span></li>
+            <li><span>Regression Testing</span></li>
+          </ul>
+        </div>
+      </section>
+      <section class="projects" id="projects">
+        <div class="projects-title">
+          <h2 class="projects-title-text">Projects</h2>
+        </div>
+        <div class="cards">
+          <a
+            href="https://www.freecodecamp.org/learn/2022/responsive-web-design/build-a-survey-form-project/build-a-survey-form"
+            target="_blank"
+            class="card card1"
+          >
+            <img src="images/fcc-project.jpg" alt="FCC Survey Form" />
+            <p>FreeCodeCamp Survey Form</p>
+          </a>
+          <a
+            href="https://guzideguzelbey.github.io/redi-project-alumniwebsite/"
+            target="_blank"
+            class="card card2"
+          >
+            <img src="images/redialumni.jpg" alt="Redi School Alumni Website" />
+          </a>
+          <a
+            href="https://guzideguzelbey.github.io/redi-GFM-react-seekfestival/"
+            target="_blank"
+            class="card card3"
+          >
+            <img src="images/seekfestival.jpg" alt="Seek Festival Website" />
+          </a>
+        </div>
+      </section>
+      <!--<section class="experience" id="experience">Experience</section>
+      <section class="education" id="education">Education</section>-->
+    </main>
+    <script>
```

Comment:
please create another file for javascript for script code

Reply:
Yeaps, I will do that.